src/utils/EquipmentType.js

```diff
@@ -8,9 +8,9 @@ import match from 'autosuggest-highlight/match';
 import parse from 'autosuggest-highlight/parse';
 import clsx from 'clsx';
 import React from 'react';
+import { LIGHT_THEME } from '../components/TopBar/TopBar';
```

Comment:
Not sure we should create  direct dependency to TopBar component code here.
Maybe use :    
`import { LIGHT_THEME } from '../..';`    
which is a dependency to the commons-ui exports index (yes I know its exports from TopBar components but, you know what I mean...)
Actually we will adapt this later...

Reply:
avoid import from '../..', it has terrible impacts over console warnings... don't know why.